packages/eas-cli/src/build/runBuildAndSubmit.ts

```diff
@@ -242,6 +247,32 @@ export async function runBuildAndSubmitAsync(
   const haveAllBuildsFailedOrCanceled = builds.every(
     build => build?.status && [BuildStatus.Errored, BuildStatus.Canceled].includes(build?.status)
   );
+
+  const simBuilds = builds.filter(
+    build =>
+      build?.status === BuildStatus.Finished &&
+      !!build?.artifacts?.applicationArchiveUrl &&
+      (build?.distribution === DistributionType.Simulator ||
+        !build.artifacts.applicationArchiveUrl.endsWith('aab'))
```

Comment:
I feel like those checks should be extracted to some functions. Aren't they used in other places? If so, it even makes more sense.